src/index.html

```diff
@@ -5,9 +5,55 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Space+Grotesk:wght@400;700&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+    >
+
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card"data-qa="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="Apple"
+      >
+      <h1 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <p class="card__id"> Product code: 195434 </p>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__rating-reviews">Reviews: 5</p>
+      </div>
+
+      <p class="card__price">
+        <span class="card__price-title">Price:</span>
+        <span class="card__price-cost">$2,199</span>
```

Comment:
I think 'value' would be a better name
```suggestion
        <span class="card__price-value">$2,199</span>
```